src/server/helpers/token.js

```diff
@@ -0,0 +1,41 @@
+import jwt from 'jsonwebtoken'
+import crypto from 'crypto'
+import { response } from '../helpers/errorHandler'
+import dotenv from 'dotenv'
+
+dotenv.config()
+
+const { SECRET } = process.env
+const tokenLife = '15m'
+
+export const generate = (payload, life = tokenLife) => {
```

Comment:
Better to use more informative names, for eg, generate what?
Refresh what?
Access what or access to??

Reply:
renamade